picard/tagger.py

```diff
@@ -842,6 +842,66 @@ def finished(file, result):
             file.set_pending()
             self._acoustid.fingerprint(file, partial(finished, file))
 
+    def extract_and_submit_acousticbrainz_features(self, objs):
+        """Extract AcousticBrainz features and submit them."""
+
+        def abz_extractor_callback(file, result, error):
+            file.clear_pending()
+
+            from concurrent.futures import Future
```

Comment:
We usually do imports at top of the file (as recommended by PEP8).

Reply:
Should be better now. :)